README.md

```diff
@@ -194,29 +128,28 @@ add
 to the cmake command.
 Release builds execute quickly.
 
-### Build F18
+### Build Flang
```

Comment:
Do we want to note that this is the out-of-tree build?

Reply:
Done!